updated vendor name / added Vanana and Waaffle keyboards

noclew opened the pull request:
<!--- Provide a general summary of your changes in the title above. -->
Updated vendor name / added Vanana and Waaffle keyboards
<!--- This template is entirely optional and can be removed, but is here to help both you and us. -->
<!--- Anything on lines wrapped in comments like these will not show up in the final text. -->

## Description

I updated the vendor name and added two Vanana and Waaffle keyboards. The other keyboards were just moved to the new vendor space, and files associated with other keyboards have not changed since the last merge. 

## Types of Changes

<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

fauxpark commented on .vscode/settings.json:
```diff
@@ -17,10 +17,14 @@
         "type_traits": "c",
         "utility": "c",
-        "ranges": "c"
+        "ranges": "c",
+        "stack": "c",
+        "ratio": "c",
+        "exception": "c"
```
```suggestion
        "ranges": "c"
```

noclew replied:
updated.

fauxpark commented on .vscode/settings.json:
```diff
@@ -17,10 +17,14 @@
         "editor.trimAutoWhitespace": false,
         "files.trimTrailingWhitespace": false
-    }
+    },
+    "C_Cpp.dimInactiveRegions": false
```
```suggestion
    }
```

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,150 @@
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
+#define VOLUP KC__VOLUP
+#define VOLDN KC__VOLDOWN
```
```suggestion
```

noclew replied:
Can I remove these custom keycodes although they are used in keymaps?

fauxpark replied:
I've updated my suggestions to completely replace them with core keycodes.

noclew replied:
Is it not recommended use aliases? I will change as you suggest, but I'd like to know what are the cons when using aliases. I used aliases because it makes a keymap file look a bit more tidy, like KC_VOLU instead of KC_AUDIO_VOL_UP. Can you let me know why it is not recommended?

fauxpark replied:
We already have short aliases for all core keycodes.

noclew replied:
> We already have short aliases for all core keycodes.

Oh, yes. If you were referring to the QMK's default keycodes AND aliases by core keycodes, I updated all my keymaps to use aliases and removed the custom keycodes. Thanks for pointing it out. I forgot that I could use aliases.

noclew replied:
All done

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,150 @@
+     *        ------------------------------------               ------------------------------------
+     */
+
+    [_BASE] = LAYOUT_vanana( \
+    KC_GRV,     KC_1,       KC_2,       KC_3,       KC_4,       KC_5,                       KC_6,       KC_7,       KC_8,       KC_9,       KC_0,       KC_BSPC,    \
+    KC_TAB,     KC_Q,       KC_W,       KC_E,       KC_R,       KC_T,                       KC_Y,       KC_U,       KC_I,       KC_O,       KC_P,       KC_BSLASH,  \
+    KC_ESC,     KC_A,       KC_S,       KC_D,       KC_F,       KC_G,       KC_RIGHT,       KC_H,       KC_J,       KC_K,       KC_L,       KC_SCLN,    KC_ENT,     \
+    KC_LSFT,    KC_Z,       KC_X,       KC_C,       KC_V,       KC_B,       KC_LEFT,        KC_N,       KC_M,       KC_COMM,    KC_DOT,     KC_SLSH,    MT(MOD_RSFT, KC_QUOT),    \
+                KC_LCTL,    KC_LALT,    KC_LGUI,    LOWER,      KC_SPC,                     KC_SPC,     RAISE,      KC_RALT,    KC_DOWN,    KC_UP                   \
+    ),
+    [_LOWER] = LAYOUT_vanana( \
+    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,                      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,
+    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,                       _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       KC_UP,          _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       KC_DOWN,        _______,    _______,    _______,    _______,    _______,    _______,    \
+                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______                 \
+    ),
+
+    [_RAISE] = LAYOUT_vanana( \
+    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,    \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,        KC_LEFT,    KC_DOWN,    KC_UP,      KC_RIGHT,   _______,    _______,    \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,        _______,    KC_PGUP,    KC_DEL ,    KC_PGDN,    _______,    _______,    \
+                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______                 \
+    ),
+    [_ADJUST] = LAYOUT_vanana( \
+    RESET  ,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    DEBUG,      \
+    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,                    _______,    _______,    _______,    _______,    _______,    _______,    \
+    KC_CAPSLOCK,_______,    RGB_HUD,    RGB_HUI,    RGB_VAD,    RGB_VAI,    VOLUP,          _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    AU_ON  ,    AU_OFF ,    _______,    _______,    _______,    VOLDN,          _______,    _______,    _______,    _______,    _______,    _______,    \
+                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______     \
```
```suggestion
    [_BASE] = LAYOUT_vanana(
    KC_GRV,     KC_1,       KC_2,       KC_3,       KC_4,       KC_5,                       KC_6,       KC_7,       KC_8,       KC_9,       KC_0,       KC_BSPC,
    KC_TAB,     KC_Q,       KC_W,       KC_E,       KC_R,       KC_T,                       KC_Y,       KC_U,       KC_I,       KC_O,       KC_P,       KC_BSLASH,
    KC_ESC,     KC_A,       KC_S,       KC_D,       KC_F,       KC_G,       KC_RIGHT,       KC_H,       KC_J,       KC_K,       KC_L,       KC_SCLN,    KC_ENT,
    KC_LSFT,    KC_Z,       KC_X,       KC_C,       KC_V,       KC_B,       KC_LEFT,        KC_N,       KC_M,       KC_COMM,    KC_DOT,     KC_SLSH,    MT(MOD_RSFT, KC_QUOT),
                KC_LCTL,    KC_LALT,    KC_LGUI,    LOWER,      KC_SPC,                     KC_SPC,     RAISE,      KC_RALT,    KC_DOWN,    KC_UP
    ),
    [_LOWER] = LAYOUT_vanana(
    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,                      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,
    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,                       _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       KC_UP,          _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       KC_DOWN,        _______,    _______,    _______,    _______,    _______,    _______,
                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______
    ),

    [_RAISE] = LAYOUT_vanana(
    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,        KC_LEFT,    KC_DOWN,    KC_UP,      KC_RIGHT,   _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,        _______,    KC_PGUP,    KC_DEL ,    KC_PGDN,    _______,    _______,
                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______
    ),
    [_ADJUST] = LAYOUT_vanana(
    RESET  ,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    DEBUG,
    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,                    _______,    _______,    _______,    _______,    _______,    _______,
    KC_CAPSLOCK,_______,    RGB_HUD,    RGB_HUI,    RGB_VAD,    RGB_VAI,    KC_VOLU,        _______,    _______,    _______,    _______,    _______,    _______,
    _______,    AU_ON  ,    AU_OFF ,    _______,    _______,    _______,    KC_VOLD,        _______,    _______,    _______,    _______,    _______,    _______,
                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______
```

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,150 @@
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+        return false;
+        break;
```
```suggestion
```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,150 @@
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+        return false;
+        break;
```
```suggestion
```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,150 @@
+                #endif
+            }
+        return false;
+        break;
```
```suggestion
```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/vanana/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,92 @@
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
+#define VOLUP KC__VOLUP
+#define VOLDN KC__VOLDOWN
```
```suggestion
```

noclew replied:
same situation where custom keycodes are used in keymaps

fauxpark commented on keyboards/delikeeb/vanana/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,92 @@
+     *        ------------------------------------               ------------------------------------
+     */
+
+    [_BASE] = LAYOUT_vanana( \
+    KC_GRV,     KC_1,       KC_2,       KC_3,       KC_4,       KC_5,                       KC_6,       KC_7,       KC_8,       KC_9,       KC_0,       KC_BSPC,    \
+    KC_TAB,     KC_Q,       KC_W,       KC_E,       KC_R,       KC_T,                       KC_Y,       KC_U,       KC_I,       KC_O,       KC_P,       KC_BSLASH,  \
+    KC_ESC,     KC_A,       KC_S,       KC_D,       KC_F,       KC_G,       KC_RIGHT,       KC_H,       KC_J,       KC_K,       KC_L,       KC_SCLN,    KC_ENT,     \
+    KC_LSFT,    KC_Z,       KC_X,       KC_C,       KC_V,       KC_B,       KC_LEFT,        KC_N,       KC_M,       KC_COMM,    KC_DOT,     KC_SLSH,    KC_QUOT,    \
+                KC_LCTL,    KC_LALT,    KC_LGUI,    LOWER,      KC_SPC,                     KC_SPC,     RAISE,      KC_LEFT,    KC_DOWN,    KC_UP                   \
+    ),
+    [_LOWER] = LAYOUT_vanana( \
+    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,                      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,
+    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,                       _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       _______,        _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       _______,        _______,    _______,    _______,    _______,    _______,    _______,    \
+                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______                 \
+    ),
+
+    [_RAISE] = LAYOUT_vanana( \
+    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,    \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,        KC_LEFT,    KC_DOWN,    KC_UP,      KC_RIGHT,   _______,    _______,    \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,        _______,    KC_PGUP,    KC_DEL ,    KC_PGDN,    _______,    _______,    \
+                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______                 \
+    ),
+    [_ADJUST] = LAYOUT_vanana( \
+    RESET  ,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    DEBUG,      \
+    _______,    RGB_TOG,    RGBF,       RGBR,       _______,    _______,                    _______,    _______,    _______,    _______,    _______,    _______,    \
+    KC_CAPSLOCK,_______,    _______,    _______,    _______,    _______,    VOLUP,          _______,    _______,    _______,    _______,    _______,    _______,    \
+    _______,    AU_ON  ,    AU_OFF ,    _______,    _______,    _______,    VOLDN,          _______,    _______,    _______,    _______,    _______,    _______,    \
+                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______     \
+    )
```
```suggestion
    [_BASE] = LAYOUT_vanana(
    KC_GRV,     KC_1,       KC_2,       KC_3,       KC_4,       KC_5,                       KC_6,       KC_7,       KC_8,       KC_9,       KC_0,       KC_BSPC,
    KC_TAB,     KC_Q,       KC_W,       KC_E,       KC_R,       KC_T,                       KC_Y,       KC_U,       KC_I,       KC_O,       KC_P,       KC_BSLASH,
    KC_ESC,     KC_A,       KC_S,       KC_D,       KC_F,       KC_G,       KC_RIGHT,       KC_H,       KC_J,       KC_K,       KC_L,       KC_SCLN,    KC_ENT,
    KC_LSFT,    KC_Z,       KC_X,       KC_C,       KC_V,       KC_B,       KC_LEFT,        KC_N,       KC_M,       KC_COMM,    KC_DOT,     KC_SLSH,    KC_QUOT,
                KC_LCTL,    KC_LALT,    KC_LGUI,    LOWER,      KC_SPC,                     KC_SPC,     RAISE,      KC_LEFT,    KC_DOWN,    KC_UP
    ),
    [_LOWER] = LAYOUT_vanana(
    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,                      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,
    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,                       _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       _______,        _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       _______,        _______,    _______,    _______,    _______,    _______,    _______,
                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______
    ),

    [_RAISE] = LAYOUT_vanana(
    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,                    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,        KC_LEFT,    KC_DOWN,    KC_UP,      KC_RIGHT,   _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,        _______,    KC_PGUP,    KC_DEL ,    KC_PGDN,    _______,    _______,
                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______
    ),
    [_ADJUST] = LAYOUT_vanana(
    RESET  ,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    DEBUG,
    _______,    RGB_TOG,    RGBF,       RGBR,       _______,    _______,                    _______,    _______,    _______,    _______,    _______,    _______,
    KC_CAPSLOCK,_______,    _______,    _______,    _______,    _______,    KC_VOLU,        _______,    _______,    _______,    _______,    _______,    _______,
    _______,    AU_ON  ,    AU_OFF ,    _______,    _______,    _______,    KC_VOLD,        _______,    _______,    _______,    _______,    _______,    _______,
                _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______
    )
```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/vanana/readme.md:
````diff
@@ -0,0 +1,24 @@
+    make delikeeb/vanana/rev2:default
+
+Flashing example for this keyboard:
+* Pro Micro (default)
+```
+    make delikeeb/vanana/rev2:default:flash
+```
+* Elite-C
+```
+    make delikeeb/vanana/rev2:default:dfu
+```
````
```suggestion

    make delikeeb/vanana/rev2:default:flash

```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/vanana/rev1/config.h:
```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID    0x9906
```
Stuff that is common to all revisions should be placed in the top level `config.h`.

noclew replied:
Thank you. I will make changes accordingly.

noclew replied:
Done!

fauxpark commented on keyboards/delikeeb/vanana/rev1/rules.mk:
```diff
@@ -0,0 +1,23 @@
+# MCU name
```
And same for `rules.mk`.

noclew replied:
Thank you. I will make changes accordingly.

fauxpark commented on keyboards/delikeeb/vaneelaex/v/config.h:
```diff
@@ -0,0 +1,162 @@
+/* defined by default; to change, uncomment and set to the combination you want */
+// #define IS_COMMAND() (get_mods() == MOD_MASK_SHIFT)
+
+/* control how magic key switches layers */
+//#define MAGIC_KEY_SWITCH_LAYER_WITH_FKEYS  true
+//#define MAGIC_KEY_SWITCH_LAYER_WITH_NKEYS  true
+//#define MAGIC_KEY_SWITCH_LAYER_WITH_CUSTOM false
+
+/* override magic key keymap */
+//#define MAGIC_KEY_SWITCH_LAYER_WITH_FKEYS
+//#define MAGIC_KEY_SWITCH_LAYER_WITH_NKEYS
+//#define MAGIC_KEY_SWITCH_LAYER_WITH_CUSTOM
+//#define MAGIC_KEY_HELP           H
+//#define MAGIC_KEY_HELP_ALT       SLASH
+//#define MAGIC_KEY_DEBUG          D
+//#define MAGIC_KEY_DEBUG_MATRIX   X
+//#define MAGIC_KEY_DEBUG_KBD      K
+//#define MAGIC_KEY_DEBUG_MOUSE    M
+//#define MAGIC_KEY_VERSION        V
+//#define MAGIC_KEY_STATUS         S
+//#define MAGIC_KEY_CONSOLE        C
+//#define MAGIC_KEY_LAYER0         0
+//#define MAGIC_KEY_LAYER0_ALT     GRAVE
+//#define MAGIC_KEY_LAYER1         1
+//#define MAGIC_KEY_LAYER2         2
+//#define MAGIC_KEY_LAYER3         3
+//#define MAGIC_KEY_LAYER4         4
+//#define MAGIC_KEY_LAYER5         5
+//#define MAGIC_KEY_LAYER6         6
+//#define MAGIC_KEY_LAYER7         7
+//#define MAGIC_KEY_LAYER8         8
+//#define MAGIC_KEY_LAYER9         9
+//#define MAGIC_KEY_BOOTLOADER     B
+//#define MAGIC_KEY_BOOTLOADER_ALT ESC
+//#define MAGIC_KEY_LOCK           CAPS
+//#define MAGIC_KEY_EEPROM         E
+//#define MAGIC_KEY_EEPROM_CLEAR   BSPACE
+//#define MAGIC_KEY_NKRO           N
+//#define MAGIC_KEY_SLEEP_LED      Z
+
```
```suggestion
```

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,134 @@
+    BASE,
+    LOWER,
+    RAISE,
+    RGBF,
+    RGBR,
+    VOLUP,
+    VOLDN
```
```suggestion
```

noclew replied:
I am using these four custom keycodes in the keymaps. Isn't it that the keymap will not work if I remove these lines?

fauxpark replied:
I'm not sure why these need to be custom keycodes, they seem to just be defined as already existing RGB and volume keycodes...

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,134 @@
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
+#define VOLUP KC__VOLUP
+#define VOLDN KC__VOLDOWN
```
```suggestion
```

noclew replied:
same situation where custom keycodes are used in keymaps

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,134 @@
+ * | Enter|   .  |   0  |   0  || Ctrl | Ctrl | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ *-------------------------------------------------------------------------------------------------------------------
+ */
+    [_BASE] = LAYOUT( \
+    KC_KP_ASTERISK, KC_KP_SLASH,    KC_KP_EQUAL,    KC_DEL,     KC_GRV,     KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,   \
+    KC_KP_PLUS,     KC_7,           KC_8,           KC_9,       KC_TAB,     KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,  \
+    KC_KP_PLUS,     KC_4,           KC_5,           KC_6,       KC_ESC,     KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_ENT,   \
+    KC_ENT,         KC_1,           KC_2,           KC_3,       KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, MT(MOD_RSFT, KC_QUOT),  \
+    KC_ENT,         KC_KP_DOT,      KC_0,           KC_0,       KC_LCTL,    KC_LCTL, KC_LALT, KC_LGUI, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT   \
+    ),
+    [_LOWER] = LAYOUT( \
+    _______,    _______,    _______,    _______,    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,    \
+    _______,    _______,    _______,    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,       _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______    \
+    ),
+    [_RAISE] = LAYOUT( \
+    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,     \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_LEFT,    KC_DOWN,    KC_UP,      KC_RGHT,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______    \
+    ),
+    [_ADJUST] = LAYOUT( \
+    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,     \
+    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    KC_CAPSLOCK,_______,    RGB_HUD,    RGB_HUI,    RGB_VAD,    RGB_VAI,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______    \
```
```suggestion
    [_BASE] = LAYOUT(
    KC_KP_ASTERISK, KC_KP_SLASH,    KC_KP_EQUAL,    KC_DEL,     KC_GRV,     KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,
    KC_KP_PLUS,     KC_7,           KC_8,           KC_9,       KC_TAB,     KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
    KC_KP_PLUS,     KC_4,           KC_5,           KC_6,       KC_ESC,     KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_ENT,
    KC_ENT,         KC_1,           KC_2,           KC_3,       KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, MT(MOD_RSFT, KC_QUOT),
    KC_ENT,         KC_KP_DOT,      KC_0,           KC_0,       KC_LCTL,    KC_LCTL, KC_LALT, KC_LGUI, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
    ),
    [_LOWER] = LAYOUT(
    _______,    _______,    _______,    _______,    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,
    _______,    _______,    _______,    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,       _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
    ),
    [_RAISE] = LAYOUT(
    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_LEFT,    KC_DOWN,    KC_UP,      KC_RGHT,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
    ),
    [_ADJUST] = LAYOUT(
    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,
    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    KC_CAPSLOCK,_______,    RGB_HUD,    RGB_HUI,    RGB_VAD,    RGB_VAI,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,86 @@
+    BASE,
+    LOWER,
+    RAISE,
+    RGBF,
+    RGBR,
+    VOLUP,
+    VOLDN
```
```suggestion
```

noclew replied:
Same here

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,86 @@
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
+#define VOLUP KC__VOLUP
+#define VOLDN KC__VOLDOWN
```
```suggestion
```

noclew replied:
and here

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,86 @@
+ * | Enter|   .  |   0  |   0  || Ctrl | Ctrl | Alt  | GUI  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ *-------------------------------------------------------------------------------------------------------------------
+ */
+   [_BASE] = LAYOUT( \
+    KC_KP_ASTERISK, KC_KP_SLASH,    KC_KP_EQUAL,    KC_DEL,     KC_GRV,     KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,   \
+    KC_KP_PLUS,     KC_7,           KC_8,           KC_9,       KC_TAB,     KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,  \
+    KC_KP_PLUS,     KC_4,           KC_5,           KC_6,       KC_ESC,     KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_ENT,   \
+    KC_ENT,         KC_1,           KC_2,           KC_3,       KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, MT(MOD_RSFT, KC_QUOT),  \
+    KC_ENT,         KC_KP_DOT,      KC_0,           KC_0,       KC_LCTL,    KC_LCTL, KC_LALT, KC_LGUI, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT   \
+    ),
+    [_LOWER] = LAYOUT( \
+    _______,    _______,    _______,    _______,    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,    \
+    _______,    _______,    _______,    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,       _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______    \
+    ),
+    [_RAISE] = LAYOUT( \
+    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,     \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_LEFT,    KC_DOWN,    KC_UP,      KC_RGHT,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______    \
+    ),
+    [_ADJUST] = LAYOUT( \
+    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,     \
+    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    KC_CAPSLOCK,_______,    RGB_HUD,    RGB_HUI,    RGB_VAD,    RGB_VAI,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,   \
+    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______    \
```
```suggestion
   [_BASE] = LAYOUT(
    KC_KP_ASTERISK, KC_KP_SLASH,    KC_KP_EQUAL,    KC_DEL,     KC_GRV,     KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_DEL,
    KC_KP_PLUS,     KC_7,           KC_8,           KC_9,       KC_TAB,     KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_BSPC,
    KC_KP_PLUS,     KC_4,           KC_5,           KC_6,       KC_ESC,     KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_ENT,
    KC_ENT,         KC_1,           KC_2,           KC_3,       KC_LSFT,    KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, MT(MOD_RSFT, KC_QUOT),
    KC_ENT,         KC_KP_DOT,      KC_0,           KC_0,       KC_LCTL,    KC_LCTL, KC_LALT, KC_LGUI, LOWER,   KC_SPC,  KC_SPC,  RAISE,   KC_LEFT, KC_DOWN, KC_UP,   KC_RGHT
    ),
    [_LOWER] = LAYOUT(
    _______,    _______,    _______,    _______,    KC_F1,      KC_F2,      KC_F3,      KC_F4,      KC_F5,      KC_F6,      KC_F7,      KC_F8,      KC_F9,      KC_F10,     KC_F11,     KC_F12,
    _______,    _______,    _______,    _______,    _______,    _______,    KC_7,       KC_8,       KC_9,       _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    KC_0,       KC_4,       KC_5,       KC_6,       _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    KC_1,       KC_2,       KC_3,       _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
    ),
    [_RAISE] = LAYOUT(
    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_HOME,    KC_BSPC,    KC_END,     _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    KC_LEFT,    KC_DOWN,    KC_UP,      KC_RGHT,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
    ),
    [_ADJUST] = LAYOUT(
    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,
    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    KC_CAPSLOCK,_______,    RGB_HUD,    RGB_HUI,    RGB_VAD,    RGB_VAI,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,
    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______
```

noclew replied:
removed back slashes

fauxpark commented on keyboards/delikeeb/waaffle/readme.md:
````diff
@@ -0,0 +1,24 @@
+    make waaffle/rev3:default
+
+Flashing example for this keyboard:
+* Pro Micro (default)
+```
+    make delikeeb/waaffle/rev3:default:flash
+```
+* Elite-C
+```
+    make delikeeb/waaffle/rev3:default:dfu
+```
````
```suggestion

    make delikeeb/waaffle/rev3:default:flash

```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/waaffle/rev3/rules.mk:
```diff
@@ -0,0 +1,22 @@
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
```
```suggestion
# Build Options
#   change yes to no to disable
#
```

noclew replied:
suggested change has been committed.

fauxpark commented on keyboards/delikeeb/vaneelaex/v/config.h:
```diff
@@ -0,0 +1,162 @@
+/*
```
Everything in this `v` folder should be moved one level up.

noclew replied:
Ahh.. this was a mistake while I moving things around. This will be deleted.

noclew replied:
v folder is deleted in the latest commit.

noclew reviewed (commented):
most suggested changes were committed with a few questions.

fauxpark commented on keyboards/delikeeb/waaffle/readme.md:
```diff
@@ -0,0 +1,19 @@
+
+Make example for this keyboard (after setting up your build environment):
+
+    make waaffle/rev3:default
```
```suggestion
    make delikeeb/waaffle/rev3:default
```

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,86 @@
+    ),
+    [_ADJUST] = LAYOUT(
+    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,
+    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,
```
```suggestion
    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGB_MOD,    RGB_RMOD,   RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,
```

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/via/keymap.c:
```diff
@@ -0,0 +1,86 @@
+    VOLDN
+};
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
```
```suggestion
```

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,133 @@
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+        return false;
+        break;
```
```suggestion
```

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,133 @@
+                update_tri_layer(_LOWER, _RAISE, _ADJUST);
+            }
+        return false;
+        break;
```
```suggestion
```

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,133 @@
+    ),
+    [_ADJUST] = LAYOUT(
+    _______,    _______,    _______,    _______,    RESET,      _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    DEBUG,
+    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,
```
```suggestion
    _______,    _______,    _______,    _______,    _______,    RGB_TOG,    RGB_MOD,    RGB_RMOD,   RGB_SAD,    RGB_SAI,    _______,    _______,    _______,    _______,    _______,    _______,
```

noclew replied:
Done!

fauxpark commented on keyboards/delikeeb/waaffle/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,133 @@
+    VOLDN
+};
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
```
```suggestion
```

fauxpark commented on keyboards/delikeeb/vaneelaex/v/readme.md:
```diff
@@ -0,0 +1,20 @@
+
+Make example for this keyboard (after setting up your build environment):
+
+    make nckiibs/vaneelaex:default
+
+Flashing example for this keyboard:
+    
+    make nckiibs/vaneelaex:default:flash
```
```suggestion
    make delikeeb/vaneelaex:default

Flashing example for this keyboard:
    
    make delikeeb/vaneelaex:default:flash
```
Should change these on the boards you've moved as well.

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,146 @@
+    BASE,
+    LOWER,
+    RAISE,
+    RGBF,
+    RGBR,
+    VOLUP,
+    VOLDN
```
```suggestion
```

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,146 @@
+    VOLDN
+};
+
+#define RGBF RGB_MODE_FORWARD
+#define RGBR RGB_MODE_REVERSE
```
```suggestion
```

fauxpark commented on keyboards/delikeeb/vanana/keymaps/default/keymap.c:
```diff
@@ -0,0 +1,146 @@
+    ),
+    [_ADJUST] = LAYOUT_vanana(
+    RESET  ,    _______,    _______,    _______,    _______,    _______,                    _______,    _______,    _______,    _______,    _______,    DEBUG,
+    _______,    RGB_TOG,    RGBF,       RGBR,       RGB_SAD,    RGB_SAI,                    _______,    _______,    _______,    _______,    _______,    _______,
```
```suggestion
    _______,    RGB_TOG,    RGB_MOD,    RGB_RMOD,   RGB_SAD,    RGB_SAI,                    _______,    _______,    _______,    _______,    _______,    _______,
```

noclew reviewed (commented):
All the suggested changes have been made.

fauxpark commented on keyboards/delikeeb/waaffle/readme.md:
```diff
@@ -0,0 +1,19 @@
+
+Flashing example for this keyboard:
+
+    make waaffle/rev3:default:flash
```
```suggestion
    make delikeeb/waaffle/rev3:default:flash
```

noclew replied:
Uhhh another miss.. Fixed!

noclew replied:
Done!

noclew reviewed (commented):
All suggestions were commited

noclew commented:
@fauxpark Hi Ryan, I think I fixed everything you suggested, but the PR page says that there is one more request to resolve. I checked all the comments, but could not locate one. Could you let me know where is that one remaining request? Thanks for helping me out!

zvecr commented:
"Breaking change" due to the folder relocation, will have to target `develop`. You might want to split out the new keyboards from the folder move, so that part can go directly to `master`.

noclew commented:
> "Breaking change" due to the folder relocation, will have to target `develop`. You might want to split out the new keyboards from the folder move, so that part can go directly to `master`.

Thanks for the reply. Can you please elaborate a bit more about the process? Do you mean that I need to: 

1. Open PR request to the **master** branch with new keyboards, without changing the folder(vendor) name.
2. Then, open another PR request to the **develop** branch, only with changing the folder(vendor) name.

Am I understanding correctly?

noclew commented:
> "Breaking change" due to the folder relocation, will have to target `develop`. You might want to split out the new keyboards from the folder move, so that part can go directly to `master`.

@fauxpark, @zvecr Hi, I am not too sure if I understood your suggestions correctly, but I reverted the folder name change and added new keyboards. I will make another request targeting to develop branch with the vendor name change. 

Let me know if this is the right procedure.
